pkg/android/process-uploads.go

```diff
@@ -29,12 +42,8 @@ func UploadAndroidNdk(fileList []string, apiKey string, applicationId string, ve
 		err = server.ProcessFileRequest(endpoint+"/ndk-symbol", uploadOptions, fileFieldData, timeout, retries, file, dryRun)
 
 		if err != nil {
-			if numberOfFiles > 1 {
-				if failOnUploadError {
-					return err
-				} else {
-					log.Warn(err.Error())
-				}
+			if strings.Contains(err.Error(), "409") && strings.Contains(err.Error(), "duplicate") {
```

Comment:
Do you have any reason to believe it could return a 409 for any other reason? If not, I think a check for status code alone would be sufficient.

Reply:
I do not but I thought the added check would ensure that we are sure its a duplicate.

Reply:
The intent of 409 being to mark the file as duplicate is clear – I'd rather this just checked the code and not introduce potential future problems if the less-defined message ever changes.